### PR TITLE
mgr/alerts: added emojis to status message

### PR DESCRIPTION
**Pull request by BlackLotus**
This commit adds the important function to add emojis/emoticons to your alert mails.
You can define emojis for every HEALTH_ status.
I even added a failsafe for unknown status (if a new HEALTH_ status is added or the status can't be pulled)
Now you can finally easily see what the status of your cluster is without parsing Text.
Default is off.

Nobody asked for this but my wife. She assured me that they just didn't know that they need it.

### CODE REVIEW

**Comment by BlackLotus** [comment]
Yeah why not? Thought about actually adding some functionality in the future.
I don't really use graphana/prometheus anymore since ceph-ansible recently broke it in our setup and we didn't yet get to fix it.

**Comment by sebastian-philipp** [comment]
> Yeah why not? Thought about actually adding some functionality in the future.
> I don't really use graphana/prometheus anymore since ceph-ansible recently broke it in our setup

did you report that?

**Comment by BlackLotus** [comment]
We use the Red Hat provided ceph-ansible and I think my colleague actually reported that.
We did the last ceph-ansible still on rh7 and recently updated to rh8.
We will soon let ceph-ansible run again against a rh8 base and if it again fails we will make another bug report. Update is scheduled for "seomtime this month". Like I said we don't really have the time. (and yes I wrote this patch in my free time :p)

**Comment by BlackLotus** [comment]
It annoys me a bit that it could in theory throw an exception if status is something not in HEALTH_OK, ERR, WRN.
Should I add an if status in emojis to check for this?
Funny writing those lines :D

File: src/pybind/mgr/alerts/module.py [hunk 1]
```diff
@@ -200,15 +222,25 @@ def _send_alert_smtp(self,
...
         # message
         self.log.debug('_send_alert_smtp')
+        # additional whitespace used to don't mess up formatting when disabled
+        emojis = {'HEALTH_OK': ' {}'.format(self.emoji_ok),
+                  'HEALTH_WARN': ' {}'.format(self.emoji_warn),
+                  'HEALTH_ERR': ' {}'.format(self.emoji_err)
+                 }
+        emote = ''
+        if self.emojis:
+            emote = emojis[status['status']]
```
Comment (tchaikov) [inline src/pybind/mgr/alerts/module.py:232]: could use emojis.get(..., default_value) to handle the case where the status is not one of the expected ones.

Comment (BlackLotus) [reply src/pybind/mgr/alerts/module.py]: I must be honest... I din't know you could define a default value with get... Thx pushed

**Comment by mykaul** [comment]
Worth re-opening before April 1st? ;-)

**Pull request closed without merging**